jetscape_analysis/analysis/analyze_events_base_STAT.py

```diff
@@ -176,12 +204,12 @@ def analyze_event_chunk(self, df_event_chunk):
                 self.observable_dict_event['event_weight'] = event_weight
                 self.observable_dict_event['pt_hat'] = event['pt_hat']
 
-                # Add event-wise centrality (same for all events in pre-computed hydro; varies event-by-event for real_time_hydro)
+                # Add event-wise centrality (same for all events in precomputed_hydro; varies event-by-event for real_time_hydro)
                 if self.is_AA:
                     self.observable_dict_event['centrality_min'] = self.centrality[0]
                     self.observable_dict_event['centrality_max'] = self.centrality[1]
-                    # This is trivially the same for each event for the pre-computed hydro,
-                    # but it varies for the on-the-fly case.
+
+                    # Update dynamic centrality range
```

Comment:
I think the old comment is more descriptive. They could merged together, but I think it's worth noting when it's meaningful (or not)

```suggestion
                    # Update dynamic centrality range
                    # This is trivially the same for each event for the pre-computed hydro,
                    # but it varies for the on-the-fly case.
```